### DISCUSSION TITLE
Annotating miRNAs in sc-ATAC dataset

### DISCUSSION THREAD

**Discussion by Eunice0519** [stuart-lab/signac#1537] [Q&A]
Hi all, 

I am trying to plot a violin plot out of my data using the code below, focusing on a non-coding miRNA region:

> VlnPlot(
     object = brain,
      features = "chr8-84208500-84208950",
      pt.size = 0.1,
      idents = c("W3","W9","W24"))
Error in FetchData.Seurat(object = object, vars = features, slot = slot) : 
  None of the requested variables were found: chr8-84208500-84208950

May I know how to annotate this region in my dataset?

Many thanks!

**Comment by timoast** [comment]
From the error message it looks like that peak cannot be found in your assay. I'd suggest double-checking that you're plotting the correct assay and that the peak is correct and present. If you are sure it's present, please follow up in the Seurat repo as the VlnPlot function is part of Seurat
